Updated `Django` to `3.2.20` to address `CVE-2023-36053`.
